server/SocketManager/socketmanager.js

```diff
@@ -1,3 +1,5 @@
+import { Server } from 'socket.io';
```

Comment:
The import statement is correct, but consider adding a comment to explain the purpose of the imported `Server` from 'socket.io'. This will improve code readability.
*This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*